cms/tests/test_views.py

```diff
@@ -229,19 +226,12 @@ def test_context_current_page(self):
         cache.clear()
         menu_pool.clear()
         context._standard_context_processors = None
-        # Number of queries when context processors is not enabled
-        with self.settings(**override):
-            with self.assertNumQueries(FuzzyInt(0, 12)) as context:
-                response = self.client.get("/en/plain_view/")
-                num_queries = len(context.captured_queries)
-                self.assertFalse('CMS_TEMPLATE' in response.context)
-        cache.clear()
-        menu_pool.clear()
+
         # Number of queries when context processor is enabled
         with self.settings(**original_context):
             # no extra query is run when accessing urls managed by standard
             # django applications
```

Comment:
Also here, we *were* comparing the number of queries to something else, which is now gone. Is this test still meaningful?